guides/common/modules/ref_custom-provisioning-snippet-example-for-enterprise-linux.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Custom_Provisioning_Snippet_Example_for_Enterprise_Linux_{context}"]
+= Custom Provisioning Snippet Example for {EL}
+
+You can use `Custom Post` snippets to call external APIs from within the provisioning template directly after provisioning a hosts.
+
+.Custom Post Hook Example for {EL}
```

Comment:
Should we put the name of the custom snippet to make it clear where the snippet is added?

Reply:
Added.